File: java/ex/ex1/Officer.java
public abstract class Officer {
    String name;
    boolean sex;
    String address;

    public Officer(String name, Boolean sex, String address) {
        this.name = name;
        this.sex = sex;
        this.address = address;
    }

    @Override
    public String toString() {
        return "Name: " + this.name + "\n" + "Sex: " + this.sex + "\n" + "Address: " + this.address + "\n";
    }
}

File: C#/Practice Project/PracticeProject1/Officer.cs
namespace PracticeProject1
{
	internal class Officer
	{
		public string Name { get; set; }
		public int Age { get; set; }
		public bool Sex { get; set; }
		public string Address { get; set; }

		public Officer(string name, int age, bool sex, string address)
		{
			Name = name;
			Age = age;
			Sex = sex;
			Address = address;
		}

		public override string ToString()
		{
			string tempS = Sex ? "male" : "female"; 
			return "\nName: " + Name +
				"\nAge: " + Age +
				"\nSex: " + tempS + 
				"\nAddress: " + Address;
		}
	}
}